src/snap.py

```diff
@@ -0,0 +1,94 @@
+#!/usr/bin/env python3
```

Comment:
@zmraul 

Accidentally deleted your comment. In response:

I don't think so. The reason `kafka_snap` was a lib was that it was being used by 2 charms. 

As the new `snap.py` here is just a very simple wrapper for `start`, `stop` and `restart` with an `install` method, it's pretty specific to ZooKeeper now.